tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -385,8 +385,8 @@ def test_derive_code_mappings_different_roots_php(self):
             }
             derive_code_mappings(self.project.id, self.event_data)
             code_mapping = RepositoryProjectPathConfig.objects.all()[0]
-            assert code_mapping.stack_root == "sentry/"
-            assert code_mapping.source_root == "src/sentry/"
+            assert code_mapping.stack_root == ""
+            assert code_mapping.source_root == "src/"
```

Comment:
This expected behavior is INCORRECT and has been incorrect since code-mappings was introduced. It is due to the erroneous definition of "packaged" languages. Normally I would not let an erroneous test case get merged, but in this case I think it may be the right idea because:

1. Fixing it requires sweeping changes to code-mapping logic that is out of the scope of this PR 
2. I am working on these sweeping changes as we speak, and they should come out quickly after this PR is merged 
3. This bug has existed ever since code-mappings has been released. This test just happens to bring more visibility to it.